### ISSUE TITLE
Revert HGG subtyping module 00 notebook to use v17

### ISSUE THREAD

**Issue by cbethell** [AlexsLemonade/OpenPBTA-analysis#900]
#### Background:

Upon digging into #877, it was found that the [`molecular-subtyping-HGG/00-HGG-select-pathology-dx.Rmd`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-HGG/00-HGG-select-pathology-dx.Rmd) file was updated to use v18 files ([line 37](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/53c1f7c8534d32da98c20f8e9136899e7dfcce92/analyses/molecular-subtyping-HGG/00-HGG-select-pathology-dx.Rmd#L37)) in merged PR #869.

However, the module's `README.md` file currently reads as follows:

_"00-HGG-select-pathology-dx.Rmd is not run via this shell script, as it should be run locally, tied to release-v17-20200908, and should not be re-rendered when there are changes to the underlying pbta-histologies.tsv file in future releases"_

#### Suggested Next Steps

This means one of two things:

1. The `README.md` file should be updated to reflect the updates made to `00-HGG-select-pathology-dx.Rmd ` 
_OR_
2. We should revert to using v17 in `00-HGG-select-pathology-dx.Rmd ` based on the original intention described in the `README.md`

I am not sure which step is most appropriate here so please feel free to weigh in!

**Comment by jharenza** [comment]
I believe we will want 
>The README.md file should be updated to reflect the updates made to 00-HGG-select-pathology-dx.Rmd
from now on

**Comment by jaclyn-taroni** [comment]
My understanding of the purpose of that notebook was to illustrate why we made the choices we did at the time, as that was before whatever was used for subsetting (usually `broad_histology` or `short_histology`) was downstream of molecular subtyping. So by tying it to that release, we could make note of what text in `pathology_diagnosis` and the free text field captured the same samples as what we were doing before using one of the fields.

**Comment by jharenza** [comment]
Ah, I see, so we probably want to update the readme to reflect that then.

**Comment by jaclyn-taroni** [comment]
In addition to the quote that was added above https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/900#issue-785346719, the README for this module also says the following when describing this notebook:

> [`00-HGG-select-pathology-dx.Rmd`](https://alexslemonade.github.io/OpenPBTA-analysis/analyses/molecular-subtyping-HGG/00-HGG-select-pathology-dx.nb.html) is a notebook used to explore the `pathology_diagnosis` and `pathology_free_text_diagnosis` fields in the `release-v17-20200908` version of `pbta-histologies.tsv`. Prior to `release-v17-20200908`, this module used `short_histology == "HGAT"` to identify samples to be included for subtyping. 
In future releases, the `short_histology` values will be derived from the `pathology_diagnosis` and `molecular_subtype` values (see [#748](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/748)); this module generates the latter. Thus, we needed to identify the samples to included for subtyping on the basis of the `pathology_diagnosis` and `pathology_free_text_diagnosis` values. This notebook looks at what `pathology_diagnosis` and `pathology_free_text_diagnosis` values are associated with samples where `short_histology == "HGAT"` in `release-v17-20200908`. The exact matches for inclusion in the `pathology_diagnosis` are saved in [`hgg-subset/hgg_subtyping_path_dx_strings.json`](hgg-subset/hgg_subtyping_path_dx_strings.json), which is used downstream in `02-HGG-molecular-subtyping-subset-files` to generate subset files.

So I think we want to go with this option:

> 2. We should revert to using v17 in `00-HGG-select-pathology-dx.Rmd` based on the original intention described in the `README.md`
